### DISCUSSION TITLE
애니메이션 동기화 에러

### DISCUSSION THREAD

**Discussion by Park-sechang** [naverz/zepeto-studio-kor#429] [5. Q&A]
QR코드를 통한 모바일로 "앉기" 애니메이션 테스트 중 에러를 발견했습니다.

 앉아있는 캐릭터의 모습이, 다른 모바일에서는 가만히 있는 "Idle" 상태이거나 뛰는 "run" 등의 상태로 다른 모습으로 나오네요.

이런 에러를 경험하셨거나 해결하신 분 있으실까요?

![InkedKakaoTalk_20220223_232231382_LI](https://user-images.githubusercontent.com/99107908/155338845-3ad9eed9-70a6-433a-9536-4b4a4ce31e68.jpg)
![InkedKakaoTalk_20220223_232231382_01_LI](https://user-images.githubusercontent.com/99107908/155338858-91a0e241-6b2b-4359-acac-4a4b5be3517e.jpg)

**Comment by x4cf** [comment]
동기화 기능을 모두 구현했는데 작동안하는건가요? 애니메이션 동기화를 자동으로 지원하지는 않습니다
코드를 볼 수 있다면 문제해결에 도움이 될것같아요 :D

**Reply by Park-sechang** [reply]
아니요. 애니메이션 동기화 기능을 따로 구현하지는 않았습니다.
서버에 캐릭터 상태를 따로 보내주어야 하는건가요?

**Reply by x4cf** [reply]
네 맞습니다 :)

**Reply by Park-sechang** [reply]
아 감사합니다!
할게 많네요..ㅎ
